v0/destinations/serenytics/transform.js

```diff
@@ -0,0 +1,176 @@
+const { EventType } = require("../../../constants");
+const {
+  CustomError,
+  getErrorRespEvents,
+  getSuccessRespEvents,
+  getHashFromArrayWithDuplicate,
+  ErrorMessage
+} = require("../../util");
+
+const {
+  CONFIG_CATEGORIES,
+  SERENYTICS_TRACK_EXCLUSION_LIST,
+  SERENYTICS_IDENTIFY_EXCLUSION_LIST,
+  SERENYTICS_PAGE_SCREEN_EXCLUSION_LIST
+} = require("./config");
+const {
+  payloadBuilder,
+  storageUrlResponseBuilder,
+  responseBuilder
+} = require("./utils");
+
+const trackResponseBuilder = (message, { Config }, payload) => {
+  const STORAGE_URL = Config.storageUrlTrack;
+  const storageUrlEventMapping = getHashFromArrayWithDuplicate(
+    Config.eventToStorageUrlMap,
+    "from",
+    "to",
+    false
+  );
+  const { event } = message;
+  if (!event) {
+    throw new CustomError(
+      `[Serenytics]: event name is required in track call.`,
+      400
+    );
+  }
+  if (!storageUrlEventMapping[event] && !STORAGE_URL) {
```

Comment:
Will it be better if we use this condition?
```suggestion
  if ( !STORAGE_URL || !storageUrlEventMapping[event]) {
```

Reply:
No, if both conditions failed then only we will throw an error, so `||` cond. will not work here.